config/webpack.config.js

```diff
@@ -323,7 +323,11 @@ module.exports = function (
 							use: getStyleLoaders({
 								importLoaders: 1,
 								modules: {
-									getLocalIdent
+									getLocalIdent,
+									// Options to restore 6.x behavior:
+									// https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md#700-2024-04-04
+									namedExport: false,
+									exportLocalsConvention: 'as-is'
```

Comment:
First of all, I'd like to know why we should do like this. Please comment on the ticket.
If we do need to do this, why should we add this repeatly in this file?
Why don't we just add it from `getStyleLoaders` function? something like...
```
cssLoaderOptions?.modules.namedExport = false;
cssLoaderOptions?.modules.exportLocalsConvention = 'as-is';
```
Or
```
const mergedCssLoaderOptions = {
	...cssLoaderOptions,
	modules: {
		...cssLoaderOptions.modules,
		namedExport: false,
		exportLocalsConvention: 'as-is'
	}
};
```
I prefer the latter since I don't want to mutable the parameter itself.

Reply:
I wrote a reason in `Additional considerations`.
And, I first tried adding modules options to `getStyleLoaders`.
However, when the method is used, the modules are overridden, so the added options are lost.
Therefore, I had to add options to each module one by one.